Makefile

```diff
@@ -8,7 +8,7 @@ all:
 	@echo "Please choose a task."
 
 test:
```

Comment:
@OskarStark In this case, just do `phpunit`?

Personally  I never use `make test`.